src/FunderMaps.Core/Entities/Building.cs

```diff
@@ -37,27 +38,33 @@ public Building()
         public bool IsActive { get; set; } = true;
 
         /// <summary>
-        ///     Address identifier.
+        ///     Geometry.
         /// </summary>
         [Required]
-        public string Address { get; set; }
+        public string Geometry { get; set; }
```

Comment:
Maybe do something with the geometry format in some way, indicating whether it's some form of JSON, hex encoding etc?

Reply:
Sure, I think it's best to create a new data type, but later